src/components/cylc/analysis/TimeSeries.vue

```diff
@@ -0,0 +1,441 @@
+<!--
+Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+  <Teleport
+    v-if="sortInputTeleportTarget"
+    :to="sortInputTeleportTarget"
+  >
+    <div class="d-flex flex-grow-1 col-gap-1">
+      <v-autocomplete
+        multiple
+        chips
+        closable-chips
```

Comment:
I'm not sure displaying the selected tasks as chips in the `v-autocomplete` is working out. And the selected tasks are already shown in the chart legend, so why not just display the number of selected tasks in the `v-autocomplete` instead of the tasks themselves?

![image](https://github.com/cylc/cylc-ui/assets/61982285/a3eeeb6f-64b1-431a-9969-51033d96c781)

Reply:
Yup, that seems like a valid UI issue. We're going to be continuously developing / maintaining this for the foreseeable. Can we create a ticket for this to be done later? Just for the sake of getting it on trunk. We already have a lot of features / requirements / tweaks further down the line.

Reply:
Bumped to https://github.com/cylc/cylc-ui/issues/1768